cmd/server-main.go

```diff
@@ -279,6 +286,9 @@ func serverMain(ctx *cli.Context) {
 	// Set uptime time after object layer has initialized.
 	globalBootTime = UTCNow()
 
+	// Enable logging when server startup phase is successful
+	logger.Disable = false
```

Comment:
As I said previously this meaning shouldn't be changed, a default value for `logger` package should be disabled is false not true. It is unusual to expect logger.Disable as true for a package. it doesn't communicate correctly as an API.